src/components/Comments.js

```diff
@@ -40,27 +43,67 @@ const Comments = (props) => {
 
   const [viewRepliesToggled, setViewRepliesToggled] = useState([])
 
+  const [attachments, setAttachments] = useState([])
+
+  const [replyAttachments, setReplyAttachments] = useState([])
+
   useEffect(() => {
-    const fetchComments = async () => {
+    const fetchStoryComments = async () => {
```

Comment:
Lets move `fetchStoryComments` outside the `useEffect`. This way we can call the function directly and remove the `fetchComments` state.

Reply:
I think we should not move this outside the `useEffect`, since in `fetchStories` we are calling `setComments`. Moving it outside of `useEffect` will cause the Comment.js component to re-render in a loop and would make unnecessary network requests.